assets/js/modules/pagespeed-insights/components/common/Recommendation.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Recommendation component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME, STRATEGY_MOBILE, STRATEGY_DESKTOP } from '../../datastore/constants';
+import Accordion from '../../../../components/Accordion';
+import { sanitizeHTML, markdownToHTML } from '../../../../util';
+const { useSelect } = Data;
+
+export default function Recommendation( props ) {
+	const {
+		auditID,
+		title,
+		referenceURL,
+		strategy,
+	} = props;
+
+	const stackPack = useSelect( ( select ) => select( STORE_NAME ).getStackPackDescriptions( referenceURL, strategy, auditID ) );
+	if ( ! Array.isArray( stackPack ) || ! stackPack.length ) {
+		return null;
+	}
```

Comment:
This should not get all stack pack descriptions for the audit, but only the `wordpress` one (if exists), see other comment.

Reply:
Updated.